modules/dxp/apps/analytics/analytics-reports-web/test/js/components/Detail.js

```diff
@@ -511,35 +575,45 @@ describe('Detail', () => {
 					'https://www.liferay.com/resources/ebooks/Becoming+a+Digital+Business-4+Common+Enterprise+Challenges+Conquered'
 				)
 			).toBeInTheDocument();
+			expect(getByText('125,461')).toBeInTheDocument();
 			expect(
 				getByText(
 					'https://www.liferay.com/resources/whitepapers/B2B+E-Commerce+RFP+Kit'
 				)
 			).toBeInTheDocument();
+			expect(getByText('85,485')).toBeInTheDocument();
 			expect(
 				getByText(
 					'https://www.liferay.com/resources/whitepapers/6+Tactics+to+Modernize+Your+Intranet'
 				)
 			).toBeInTheDocument();
+			expect(getByText('84,564')).toBeInTheDocument();
 			expect(
 				getByText(
 					'https://www.liferay.com/resources/case-studies/materion-case-study'
 				)
 			).toBeInTheDocument();
+			expect(getByText('5,846')).toBeInTheDocument();
 			expect(
 				getByText(
 					'https://www.liferay.com/web/l/a1-hrvatska-case-study'
 				)
 			).toBeInTheDocument();
+			expect(getByText('3,521')).toBeInTheDocument();
 
 			expect(getByText('view-more')).toBeInTheDocument();
 
 			expect(getByText('top-referring-domains')).toBeInTheDocument();
 			expect(getByText('http://youtube.com/')).toBeInTheDocument();
+			expect(getByText('17,985,230')).toBeInTheDocument();
 			expect(getByText('http://www.google.com/')).toBeInTheDocument();
+			expect(getByText('12,218,030')).toBeInTheDocument();
 			expect(getByText('http://microsoft.com/')).toBeInTheDocument();
```

Comment:
about these test "URLs", I think we usually use "example.com" but that's not a blocked for me

Reply:
Indeed, per [RFC 2606](https://tools.ietf.org/html/rfc2606#section-3).

Reply:
Ok, agreed. I'm going to change it to `example.whatever` (com, org, info, es...). What do you think? Because at least I need 5-10 URLs. What happen with the other URLs? They are from liferay.com, is there a problem with that? Thanks

Reply:
You can make as many `foo.example.com`, `bar.example.com` domains as you want. Of course, like @julien says, it's not a blocker — it's just a nice way of signalling "I'm using this as an, um, example, only, and there is nothing special about this site in itself that is relevant to the test".